rust/src/cli/service.rs

```diff
@@ -15,6 +21,12 @@ pub(crate) fn ncl_service(
         .value_of(crate::CONFIG_FOLDER_KEY)
         .unwrap_or(crate::DEFAULT_SERVICE_FOLDER);
 
+    let pin_iface_name_dir = format!("{folder}/{PIN_IFACE_NAME_FOLDER}");
+    let pin_iface_path = Path::new(&pin_iface_name_dir);
+    if pin_iface_path.exists() {
+        pin_iface_name(&pin_iface_path)?;
```

Comment:
So the opt-in mechanism here is that the user creates a file?  WDYT about my proposal of a CLI command instead in [the bug](https://bugzilla.redhat.com/show_bug.cgi?id=2182769)?

Reply:
I prefer `no issue no change` in case there is side impact on this.

Reply:
Yes. If there is `pin_iface_name` folder in `/etc/nmstate` and `pin.yml` file there, nmstate do the pin work.